frontend/app/routes/_gcweb-app.personal-information.home-address.edit.tsx

```diff
@@ -73,17 +74,37 @@ export async function action({ request }: ActionFunctionArgs) {
   const formData = Object.fromEntries(await request.formData());
   const parsedDataResult = await formDataSchema.safeParseAsync(formData);
 
+  const wsaddressWebService = await getWSAddressService();
+
   if (!parsedDataResult.success) {
     return json({
       errors: parsedDataResult.error.flatten(),
       formData: formData as Partial<z.infer<typeof formDataSchema>>,
     });
   }
+  const validateResultAddress = await wsaddressWebService.validateWSAddress({
+    addressLine: parsedDataResult.data.address,
+    city: parsedDataResult.data.city,
+    country: parsedDataResult.data.country,
+    language: 'EN',
+    postalCode: parsedDataResult.data.postalCode ?? '',
+    province: parsedDataResult.data.province ?? '',
+    geographicScope: 'GS',
+    parseType: 'PT',
+  });
+
+  console.debug('VALIDATE RETURN ' + JSON.stringify(validateResultAddress, null, 2));
```

Comment:
Please remove this. We don't want any `console` calls in the main branch. If you want to log something, use a logger (but you can only log stuff from loaders and actions).

Reply:
done